packages/apps/docs/src/docs/marmalade/what-is-marmalade.md

```diff
@@ -0,0 +1,180 @@
+---
+title: What is Marmalade?
+description: Build a non-fungible token (NFT) marketplace using Marmalade and Pact smart contracts.
+menu:  Non-fungible tokens (NFT)
+label: What is Marmalade?
+order: 2
+layout: full
+---
+
+# What is Marmalade?
+
+Marmalade is the name of the Kadena **token standard**.
+This token standard defines the interfaces for minting digital items with Kadena **smart contracts** and protecting those digital items with Kadena **token policies**.
+
+Digital items that are unique, one-of-a-kind assets are most commonly referred to as non-fungible tokens (NFTs).
+However, the Kadena token standard—influenced by the Ethereum [multi-token standard](https://ethereum.org/developers/docs/standards/tokens/erc-1155)—enables you to define, mint, and secure tokens that are fully fungible as interchangeable coins, partially fungible as a limited edition with a fixed supply, or completely non-fungible as unique items.
+Tokens can also be restricted to a single owner with rules for transferring ownership or support shared ownership with multiple potential stakeholders.
+
+As a creator, you have complete control over your token properties and the rules for minting, owning, and transferring tokens.
+
+## Traditional marketplaces for non-fungible tokens
+
+Non-fungible tokens defined using other token standards have two main drawbacks:
+
+- Most token standards assume that a change of ownership is a transfer operation. 
+  However, a transfer-based model doesn't take into account the non-transferable nature of a unique asset.
+- Most token standards don't provide any guarantees that creators will be paid royalties for their work or  what they'll be paid from the exchanges that host their work.
+
+With Marmalade, Kadena eliminates these issues by making **sales contracts** and **token policies** core components of the token standard.
+
+## Sales contracts
+
+In the simplest sense, a sale is typically a three-step process:
+
+1. A seller lists an item for sale with a price.
+
+2. A buyer agrees to pay the seller for the item at a mutually agreed-upon price.
+
+3. The seller receives payment and transfers the item to the buyer.
+
+On the Kadena network, Pact smart contracts use `pact` functions to define multi-step operations like cross-chain transfers and automatic gas payments.
```

Comment:
I'm trying to avoid using the terminology that way, at least in this introductory part, treating `pact` as a feature of the language instead of a unique entity unto itself.